config/database.yml

```diff
@@ -19,6 +19,8 @@ default: &default
   encoding: unicode
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
+  username: postgres
+  password: 1234
```

Comment:
We don't use the same local configuration. You should not upload your change to this file. It may create errors to my side.

Reply:
Got it
lemme Fix this